provider/cmd/pulumi-resource-command/schema.json

```diff
@@ -8,7 +8,8 @@
             "properties": {
                 "user": {
                     "type": "string",
-                    "descripton": "The user that we should use for the connection. Defaults to `root`."
+                    "descripton": "The user that we should use for the connection.",
+                    "default": "root"
```

Comment:
I believe with `default` that the `remoteconnection` struct can now type these as non-pointer types, which might clean up some implementation code?